contracts/lib/arbitration/IAragonAppFeesCashier.sol

```diff
@@ -3,15 +3,16 @@ pragma solidity ^0.4.24;
 import "../token/ERC20.sol";
 
 
+// Note
```

Comment:
Similar to the other `arbitration/` contracts, should we add some:

- Natspec
- Information about where this came from (e.g. aragon-court)?